### PR TITLE
add `@scope` to `<language>` by factoring a new class from `@scope` of att.handFeatures

### PR DESCRIPTION
**Pull request by sydb**
Basic idea was to address #2516 by removing the definition of `@scope` from att.handFeatures, modifying it slightly, and putting it into a new class "att.impreciseScope", then making both `<language>` and att.handFeatures members of that new class.

This was a BLOCKED **draft** PR because it would not build until #2630 had merged. (Or changes are made here to account for the lack of #2630. :-) #2630 has been merged, so this PR is good to go.

### CODE REVIEW

**Comment by ebeshero** [comment]
@sydb After I merged in PR #2630 , I came back here to rerun the test on this PR. They failed again, but at first it was pretty simple: just needed to update specs + ST to reference att.scope instead of the original att.impreciseScope. I went through and just fixed that quickly, and after that we can build the Guidelines with this branch, but `make validate` fails (you'll see it in the test on this branch). It's failing due to the new BIB entry with `<idno>` set to `type="URN"`:

```xml
 <bibl xml:id="bib4handnoteyi" xml:lang="en">
	<author>Libelt, Karol</author>.
	<title level="m" xml:lang="pl">Wykłady Humboldta na uniwersytecie Berlińskim: notaty prelekcyj tych po uczniu Jego Karolu Libelcie</title>,
	<publisher xml:lang="de">Deutsches Textarchiv</publisher>,
	<date>2024-04-02</date>.
	<idno type="URN">urn:nbn:de:kobv:b4-30960-7</idno>,
	<ptr target="https://www.deutschestextarchiv.de/book/download_xml/libelt_hs6623ii_1828"/>
	<date type="accessed" when="2024-12-04"/>
      </bibl>
```

**Comment by ebeshero** [comment]
Of course...and I bet we can fix that by updating this branch from dev now...

**Comment by ebeshero** [comment]
Huzzah! passes tests now... @sydb Think you this is ready for review?

File: P5/Source/Specs/handNote.xml [hunk 1]
```diff
@@ -26,20 +23,34 @@
...
-        <p>Written in insular
-      phase II half-uncial with interlinear Old English gloss in an Anglo-Saxon pointed
-      minuscule.</p>
-      </handNote>
+    <!-- This is a translation of "HANDNOTE-egXML-yi" -->
+    <egXML xmlns="http://www.tei-c.org/ns/Examples" xml:id="HANDNOTE-egXML-yi_en" source="#bib4handnoteyi">
+      <handNotes>
+	<handNote xml:id="yien.major_hand" scope="major" medium="brown_ink" scribe="author" scribeRef="#KL">Handwriting of the copyist Karol Libelt, brown ink, in German cursive script.</handNote>
+	<handNote xml:id="yien.pencil" scope="minor" medium="pencil" scribe="annotator">A note, pencil, unknown author.</handNote>
+	<handNote xml:id="yien.pencil2" scope="minor" medium="pencil" scribe="archivist">writer of the title page Józef Łepkowski, pencil.</handNote>
```
Comment (sabineseifert) [inline P5/Source/Specs/handNote.xml]: For the sake of being parallel to the other handNotes, "writer" should perhaps be written "Writer".

**Review by sabineseifert — APPROVED** [review]

**Comment by ebeshero** [comment]
This looks good! I implemented @sabineseifert's suggested edit, and just updated the branch from dev and tested / built Guidelines + Exemplars / validated locally. All's well. I'll merge this as soon as it passes tests here.

File: P5/Source/Specs/att.scope.xml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+        <valItem ident="minor">
+          <desc versionDate="2024-12-04" xml:lang="en">this feature is used occasionally through the document</desc>
+        </valItem>
+      </valList>
+      <exemplum xml:lang="en">
+        <egXML xmlns="http://www.tei-c.org/ns/Examples" xml:id="class-attr-scope-egXML-ls">
+          <langUsage>
+            <language ident="en" scope="major"/>
+            <language ident="es" scope="minor"/>
+            <language ident="x-ww" scope="minor">An invented language the children call <name>Wikwah</name>.</language>
```
Comment (ebeshero) [inline P5/Source/Specs/att.scope.xml:26]: This exemplum is super helpful! :-)

**Review by ebeshero — APPROVED** [review]
This all looks good. Examples are very helpful!

**Pull request merged**